public/duck/index.vwf.config.yaml

```diff
@@ -19,3 +19,4 @@ model:
   vwf/model/buzz:
```

Comment:
Is the duck by default now rendered in stereo? I'm not seeing any difference when I run locally.

Do we want to have a separate app as a stereo example?